app/pathogen/arbovirus/dashboard/page.tsx

```diff
@@ -1,25 +1,83 @@
+import { MedianSeroPrevByWHOregion } from "../analyze/recharts";
+import Filters, { FilterableField } from "./filters";
+import { CardConfiguration, CardType, getConfigurationForCard } from "@/components/ui/card-collection/card-collection-types";
+import { CardCollection } from "@/components/ui/card-collection/card-collection";
+import { useMap } from "react-map-gl";
 
 export default function ArbovirusDashboard() {
   // Need to make the visualizations dynamic. Unsure how to do this well using CSS.
-  return (
+  const allMaps = useMap();
+
+  const renderPlotCardContent = useCallback(({cardConfigurations}: {cardConfigurations: CardConfiguration[]}) => (
+    <CardContent className={"px-0 h-full flex flex-col"}>
+      <div className={"flex space-between py-4"}>
+        <h3 className="w-full text-center text-lg">
+          Median seroprevalence of arboviruses by WHO region
+        </h3>
+        <button aria-label="Close graphs">
+          <X onClick={() => {getConfigurationForCard(cardConfigurations, 'plots', CardType.EXPANDABLE).minimizeCard()}} className={"h-full"}/>
+        </button>
+      </div>
+      <MedianSeroPrevByWHOregion />
+    </CardContent>
+  ), [])
+
+  const renderMapCardContent = useCallback(({cardConfigurations}: {cardConfigurations: CardConfiguration[]}) => (
+    <ArbovirusMap
+      expandVisualizations={() => {getConfigurationForCard(cardConfigurations, 'plots', CardType.EXPANDABLE).expandCard()}}
+      areVisualizationsExpanded={getConfigurationForCard(cardConfigurations, 'plots', CardType.EXPANDABLE).isExpanded}
+    />
+  ), [])
+
+  const renderFiltersCardContent = useCallback(({ cardConfigurations }: {cardConfigurations: CardConfiguration[]}) => (
     <>
-      <Card className={"row-span-2 col-span-4 pr-4 pb-4"}>
-        <CardContent className={"px-0 h-full flex flex-col"}>
-          <h3 className="py-4 w-full text-center text-lg">
-            Median seroprevalence of arboviruses by WHO region
-          </h3>
-          <MedianSeroPrevByWHOregion />
-        </CardContent>
-      </Card>
-      <MapAndFilters />
+      <CardHeader>
+        <CardTitle>Filters</CardTitle>
+      </CardHeader>
+      <CardContent>
+        <Filters excludedFields={[FilterableField.pathogen]} />
+      </CardContent>
     </>
-  );
+  ), [])
+
+  const cardInputData = useMemo(() => [
+    {
+      order: 1,
+      cardId: 'plots',
+      type: CardType.EXPANDABLE as const,
+      isExpandedByDefault: false,
+      expandedColumnCount: 4,
+      cardClassname: "row-span-2 pr-4 pb-4",
+      renderCardContent: renderPlotCardContent
+    },
+    {
+      order: 2,
+      cardId: 'map',
+      type: CardType.FILL_REMAINING_SPACE as const,
+      cardClassname: "w-full h-full overflow-hidden row-span-2 relative",
+      renderCardContent: renderMapCardContent,
+      onCardSizeChange: () => {
+        if(allMaps['arboMap']) {
+          allMaps['arboMap'].resize();
+        }
+      }
```

Comment:
Yes this is necessary, the map bugs out if you try to resize it without also calling this method. Basically all of the gross stuff in `card-collection.tsx` is there because I'm trying to set up the order of events to happen correctly for the map resizing.

Silly stuff like this also contribute to why I was unable to find a css solution that worked and why I went towards making a component that's a bit more complicated so that we just solve this "resizable map" problem once, it's a lot more annoying than you might initially think lol